apps/frontend/core-dapp/src/stores/BalancerStore.ts

```diff
@@ -44,10 +45,11 @@ export class BalancerStore {
         this.lastCalled = Date.now() as DateTimeInMs
 
         try {
-          const price = await this.getWstEthPrice()
-          if (price !== undefined) {
+          const oneWstEth = BigNumber.from(10).pow(18)
```

Comment:
Maybe better to just use parseEther or at least have a reference to wstETH decimals?

```suggestion
          const oneWstEth = parseEther('1')
```

or 

```suggestion
          const oneWstEth = BigNumber.from(10).pow(WSTETH_DECIMALS)
```